### PR TITLE
Add support for Prometheus metrics

### PR DESCRIPTION
**Pull request by zeako**

### CODE REVIEW

File: nsqworker/nsqwriter.py [hunk 1]
```diff
@@ -81,11 +82,13 @@ def finish_pub(self, conn, data, topic, payload):
...
             # Message failed, re-send
+            MESSAGES_RETRY_COUNTER.labels(topic).inc()
             self.logger.error("Message failed, waiting {} seconds before trying again..".format(delay))
             # Take a short break and then try to resend the message
             if isinstance(payload, str):
                 self.io_loop.call_later(delay, self.send_message, topic, payload)
             elif isinstance(payload, list):
                 self.io_loop.call_later(delay, self.send_messages, topic, payload)
         else:
+            MESSAGES_SENT_COUNTER.labels(topic).inc()
```
Comment (ovaltzer) [inline nsqworker/nsqwriter.py]: don't you want to count per (channel,topic)?

Comment (zeako) [reply nsqworker/nsqwriter.py]: messages are published to topics

File: nsqworker/__init__.py [hunk 2]
```diff
@@ -0,0 +1 @@
+from . import metrics
```
Comment (azk) [inline nsqworker/__init__.py]: not sure this is needed...

**Comment by zeako** [comment]
:O

**Pull request closed without merging**